File: 1385-find-the-distance-value-between-two-arrays/1385-find-the-distance-value-between-two-arrays.java
class Solution {
    public int findTheDistanceValue(int[] arr1, int[] arr2, int d) {
        int res = 0;
        
        for(int i = 0; i < arr1.length; i++){
            int counter = 0;
            for(int j = 0; j < arr2.length; j++){
                if(Math.abs(arr1[i] - arr2[j]) > d)
                    counter++;
            }
            if(counter == arr2.length)
                res++;
        }
        return res;
    }
}

File: 1385-find-the-distance-value-between-two-arrays/1385-find-the-distance-value-between-two-arrays.swift
class Solution {
    func findTheDistanceValue(_ arr1: [Int], _ arr2: [Int], _ d: Int) -> Int {
        var val = 0
    
        var i = 0
        while i < arr1.count {
            var j = 0
            while j < arr2.count {
                if abs(arr1[i]-arr2[j]) <= d { 
                    break 
                }
                j += 1
            }
            if j == arr2.count { val += 1 }
            i += 1
        }

        return val
    }
}